procurement/templates/procurement/home.html

```diff
@@ -9,14 +9,16 @@
 <div class="bg-gray-100 border-top py-4 py-md-5">
     <div class="container" style="max-width: 44em">
 
-        <form class="row" action="" method="GET">
-            <fieldset class="col-sm-auto me-auto">
-                <label for="council-search" class="form-label">Search by postcode or council name</label>
-                <div class="search-input">
-                    <!-- { % include icons/search.html %} -->
-                    <input type="search" name="pc" id="council-search" class="form-control" value="{{ filter.form.pc.value }}">
-                </div>
-            </fieldset>
+        <form action="" method="GET">
+            <div class="row">
+                <fieldset class="col-sm-auto me-auto">
```

Comment:
A minor point, but seems odd to me that we use a `<fieldset>` here, since it’s wrapping just a single control. Usually you’d use a fieldset to group a set of related checkboxes or radios, or to create a logical "section" inside a form. I think, here, we’re really just after a `<div>`.

Reply:
One for @lucascumsille